atlasdb-client/src/main/java/com/palantir/atlasdb/keyvalue/impl/AbstractKeyValueService.java

```diff
@@ -361,4 +370,16 @@ public RowColumnRangeIterator getRowsColumnRange(TableReference tableRef,
                                                                           cellBatchHint,
                                                                           timestamp);
     }
+
+    @Override
+    public void rehydrateLoggingArgSupplier() {
```

Comment:
just sort of thinking out loud here.. does this functionality need to be a part of the KVS? it doesn't seem like anything here is KVS-specific (although all the classes have `KeyValueService` in the name)

one idea would be to just make this a static class (normally not ideal, but this is just for logging, which is something that all the classes need to do).